### ISSUE TITLE
The super resolution can't resize the image with a two-fold resolution

### ISSUE THREAD

**Issue by tucong** [kunzmi/ImageStackAlignator#23]
It's a great work! I'm a little confused, I Check the box of Super resolution and also re-clicked Prepare accumulation, but the output image still keep the same size as the original image, without a two fold pixels. Thank you very much for your answer！

**Comment by tucong** [comment]
I am sorry I have checked the result and find it achieve a two-fold resolution in the central area. Thanks! :)

**Comment by sheuchenko** [comment]
> I am sorry I have checked the result and find it achieve a two-fold resolution in the central area. Thanks! :)

hello tucong
I meet an error and have not got the final results. I have finished step1 and 2 and 3 successfully, but when clicking Prepare accumulation, it shows the exception: ManagedCuda.NPP.NPPException:“Device allocation error” .
I wonder if you ever meet this problem? Thanks~

**Comment by tucong** [comment]
> hello tucong
> I meet an error and have not got the final results. I have finished step1 and 2 and 3 successfully, but when clicking Prepare accumulation, it shows the exception: ManagedCuda.NPP.NPPException:“Device allocation error” .
> I wonder if you ever meet this problem? Thanks~

It may be that the GPU memory is not enough, you can try to load only two original images for reconstruction.

**Comment by sheuchenko** [comment]
> > hello tucong
> > I meet an error and have not got the final results. I have finished step1 and 2 and 3 successfully, but when clicking Prepare accumulation, it shows the exception: ManagedCuda.NPP.NPPException:“Device allocation error” .
> > I wonder if you ever meet this problem? Thanks~
> 
> It may be that the GPU memory is not enough, you can try to load only two original images for reconstruction.

Yes! Now I use the photos with lower resolution, it runs successfully. Thank you so much!

**Comment by iKaHibi** [comment]
> I am sorry I have checked the result and find it achieve a two-fold resolution in the central area. Thanks! :)

Hi tucong!
I am wondering how did you checked the tiff picture and found the central area to be two-fold resolution?
I am now using the PIL.Image.open package of python which can get a width×height array. I am confused of how to distinguish where it gets higher resolution.
